src/components/Card/Card.tsx

```diff
@@ -49,15 +46,15 @@ class Card extends React.Component<Props, State> {
 
   render () {
     const { windowWidth } = this.state
-    const { onPress, factor = 1, icon, title, imageUrl, minorText: minor, className, style } = this.props
+    const { onPress, factor = 1, icon, imageUrl, className, style } = this.props
     const width = this._self ? this._self.clientWidth : windowWidth / factor
+    var {title} = this.props
+    if (title.length > 16) {
```

Comment:
We may need to add logic if the icon is supplied to shorten this by a character.

Reply:
In hindsight, should we be doing this in voa-zh before we pass the title and icon to Card? This makes it truncate all languages at 16 characters, which is pretty aggressive for English, and English probably doesn't need two ellipsis, right?

Reply:
Good point, probably a good idea to move it to the app itself